bin/mod/ent-kubectl

```diff
@@ -72,9 +72,9 @@ RUN() {
       fi
 
       if [ -n "$CMD" ]; then
-        _kubectl "$CMD" -n "$NAMESPACE" "$@"
+        ENT_KUBECTL_NO_CUSTOM_ERROR_MANAGEMENT=true _kubectl "$CMD" -n "$NAMESPACE" "$@"
```

Comment:
can't we define the variable once?

Reply:
ENT_KUBECTL_NO_CUSTOM_ERROR_MANAGEMENT is an implicit parameter of _kubectl and I find this syntax more explicit